test_polars_import intermittently fails

### Issue description

One of the two intermittently failed tests mentioned in #9122.

I looked into the `test_polars_import` and I've found the issue:

```python
        if_err = f"Possible import speed regression; took {total_import_time//1_000}ms"
        assert total_import_time < 200_000, f"{if_err}\n{df_import}"
```

The total import time was taking too long. The test runs the following:

```
python -X importtime -c "import polars"
```

And looking at the output, we have this:

```
FAILED tests/unit/test_polars_import.py::test_polars_import - AssertionError: Possible import speed regression; took 205ms
  ┌───────────────────────────────────────────────────────────────┬──────────┬─────────────────┐
  │ import                                                        ┆ own_time ┆ cumulative_time │
  │ ---                                                           ┆ ---      ┆ ---             │
  │ str                                                           ┆ i32      ┆ i32             │
  ╞═══════════════════════════════════════════════════════════════╪══════════╪═════════════════╡
  │  polars                                                       ┆ 1298     ┆ 123721          │
  |                                      ...                                                   |
  │        email                                                  ┆ 469      ┆ 469             │
  │  site                                                         ┆ 64801    ┆ 205091          │
  │    sitecustomize                                              ┆ 232      ┆ 232             │
  │    _sysconfigdata_m_linux_x86_64-linux-gnu                    ┆ 734      ┆ 734             │
  │    coverage                                                   ┆ 341      ┆ 122808          │
  │      coverage.control                                         ┆ 2058     ┆ 122007          │
                   ...
```

The `site` import is separate from `polars`. Looking at the [docs for the site module](https://docs.python.org/3/library/site.html), we have this:

> **This module is automatically imported during initialization**. The automatic import can be suppressed using the interpreter’s [`-S`](https://docs.python.org/3/using/cmdline.html#cmdoption-S) option.
>
>Importing this module will append site-specific paths to the module search path and add a few builtins, unless [`-S`](https://docs.python.org/3/using/cmdline.html#cmdoption-S) was used. In that case, this module can be safely imported with no automatic modifications to the module search path or additions to the builtins. To explicitly trigger the usual site-specific additions, call the [`site.main()`](https://docs.python.org/3/library/site.html#site.main) function.

So, for our import test, I believe we should use the `-S` option, since we're only concerned with how long polars takes to load, not the `site` module, which is outside of our control.

--- a/py-polars/tests/unit/test_polars_import.py
+++ b/py-polars/tests/unit/test_polars_import.py
@@ -42,7 +42,7 @@ def _import_timings_as_frame(best_of: int) -> pl.DataFrame:
 
 @pytest.mark.skipif(sys.platform == "win32", reason="Unreliable on Windows")
 def test_polars_import() -> None:
-    # note: take the fastest of three runs to reduce noise.
+    # note: take the fastest of several runs to reduce noise.
     df_import = _import_timings_as_frame(best_of=3)
 
     with pl.Config() as cfg:
@@ -61,8 +61,9 @@ def test_polars_import() -> None:
             assert not_imported, f"{if_err}\n{df_import}"
 
         # ensure that we do not have an import speed regression.
-        total_import_time = df_import["cumulative_time"].max()
-        assert isinstance(total_import_time, int)
+        polars_import = df_import.filter(pl.col("import").str.strip() == "polars")
+        polars_import_time = polars_import["cumulative_time"].item()
+        assert isinstance(polars_import_time, int)
 
-        if_err = f"Possible import speed regression; took {total_import_time//1_000}ms"
-        assert total_import_time < 200_000, f"{if_err}\n{df_import}"
+        if_err = f"Possible import speed regression; took {polars_import_time//1_000}ms"
+        assert polars_import_time < 200_000, f"{if_err}\n{df_import}"

